### DISCUSSION TITLE
Marketplace Cloud

### DISCUSSION THREAD

**Discussion by backingcorp** [swellstores/community#127] [Help] [answered]
Hi guys! I'm interested in the Cloud hosted version of Swell but not sure if it allows the Marketplace functionality?

Will the 2% commission apply to all vendors and can I charge vendors monthly to list their products in the cloud version of Swell?

Thanks a lot
Mark

**Comment by sags95** [comment]
Hey @backingcorp , we have many marketplaces running on Swell today. The 2% fee will be for all orders placed through your store (the entire marketplace). Since Swell has native subscription functionality, you can certainly create a digital subscription product for your vendors to subscribe to in order for them to post products.

**Comment by backingcorp** [comment]
I see here: https://developers.swell.is/guides/build-a-marketplace/create-account-hierarchies#creating-new-products
and I like to know if all the Stripe connect and Netlify coding will work with Swell free edition?

Thanks
https://developers.swell.is/guides/build-a-marketplace/create-account-hierarchies#creating-a-platform-fee

**Reply by sags95** [reply, marked as answer]
Yes you should be able to do all that on the Community plan.
